spec/inputs/pipe_spec.rb

```diff
@@ -1,43 +1,63 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
Me having the old "automatic" behaviour of pointing everything to a single spec_helper per project, so there we can have specific helpers, mocks, etc. But I understand this does not provide much here :+1: